src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -77,18 +83,28 @@ public static ExportConfiguration load(BriefcasePreferences prefs) {
         prefs.nullSafeGet(START_DATE).map(LocalDate::parse),
         prefs.nullSafeGet(END_DATE).map(LocalDate::parse),
         prefs.nullSafeGet(PULL_BEFORE).map(Boolean::valueOf),
-        prefs.nullSafeGet(PULL_BEFORE_OVERRIDE).map(PullBeforeOverrideOption::from)
+        prefs.nullSafeGet(PULL_BEFORE_OVERRIDE).map(PullBeforeOverrideOption::from),
+        prefs.nullSafeGet(FORM_NEEDS_PRIVATE_KEY).map(Boolean::valueOf)
```

Comment:
This is only used to load the default configuration. We need to default this to Optional.empty().

Also, let's change the name to explicitly state that this factory is meant only for loading the default configuration.

Reply:
Ok got it